Description: Inheritate CXXFLAGS from the environment, to use dpkg-buildflags'
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-01-09
--- a/configure.ac
+++ b/configure.ac
@@ -89,7 +89,7 @@
 warning_flags="-Wall -Wno-unknown-pragmas"
 debug_flags=""
 optimization_flags="-O2"
-CXXFLAGS="${optimization_flags} ${warning_flags} ${debug_flags} ${cups_cflags}"
+CXXFLAGS+=" ${optimization_flags} ${warning_flags} ${debug_flags} ${cups_cflags}"
 
 
 AC_CHECK_LIB(cups, cupsMarkOptions, , AC_ERROR([Can't find cups library]))
